apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts

```diff
@@ -484,6 +484,16 @@ export class BuyPlanComponent implements OnInit, OnDestroy {
         this.componentStore.activeCompany$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
             if (response) {
                 this.activeCompany = response;
+                if (this.activeCompany) {
```

Comment:
If you check response and after that assign response to activeCompany and then check again activeCompany 

I think no use check twice